src/Symfony/Component/Form/FormError.php

```diff
@@ -187,4 +187,18 @@ public function unserialize($serialized)
     {
         list($this->message, $this->messageTemplate, $this->messageParameters, $this->messagePluralization, $this->cause) = unserialize($serialized);
     }
+
+    /**
+     * @see JsonSerializable::jsonSerialize()
+     */
+    public function jsonSerialize()
+    {
+        return array(
+            'message' => $this->message,
+            'messageTemplate' => $this->messageTemplate,
+            'messageParameters' => $this->messageParameters,
+            'messagePluralization' => $this->messagePluralization,
+            'cause' => $this->cause,
```

Comment:
I think you'd be better off creating your own JSON converter for form errors than relying on the framework. This way you have much more control of your API.

I don't see much purpose in this PR. How errors should be converted to JSON depends very much on the project, no?